hive/glue-sync.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source /usr/lib/hustler/bin/qubole-bash-lib.sh
+source /usr/lib/qubole/bootstrap-functions/hive/thrift-metastore.sh
+
+##
+# Installs Hive Glue Catalog Sync Agent
+# param1 - Region for AWS Athena. Defaults to us-east-1
+# Requires Hive 2.x
+#
+function install_glue_sync() {
+    aws_region=${1:-us-east-1}
+
+    is_master=$(nodeinfo is_master)
+    hive_version=$(nodeinfo hive_version)
+    if [[ ${is_master} == "1" && ${hive_version} == 2* ]]; then
```

Comment:
Nit - put 2* in quotes.

Reply:
Putting 2* in quotes will match the literal "2*". I want to match any Hive 2.x version